internal/provider/resource_kubernetes_cloud_test.go

```diff
@@ -0,0 +1,70 @@
+package provider
+
+import (
+	internaltesting "github.com/juju/terraform-provider-juju/internal/testing"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func getFakeCloudConfig() string {
```

Comment:
A fake config will not work for the Acceptance tests run against live configs. We need to find a way to get the microk8s config from the runner.